portal/forms/home.py

```diff
@@ -65,8 +65,12 @@ def clean_name(self):
 
     def clean_message(self):
         message = self.cleaned_data.get("message", None)
-        if re.match(re.compile('^[ -~]+$'), message) is None:
-            raise forms.ValidationError("Your message may not contain special characters.")
+        line_list = message.splitlines()
+
+        for line in line_list:
+            if re.match(re.compile('^[ -~]+$'), line) is None:
+                raise forms.ValidationError("Your message may not contain special characters.")
```

Comment:
Also, you should save the compiled expression somewhere so you don't recompile it at each step of the loop :)

Reply:
Not a huge deal though: https://docs.python.org/2/library/re.html#re.compile